aurelia-bootstrap-select/src/abp-select.js

```diff
@@ -228,6 +228,7 @@ export class AbpSelectCustomElement {
   collectionChangedObserver(newCollection, oldCollection) {
     setTimeout(() => {
       this.domElm.selectpicker('refresh');
+      this.renderPreSelection();
```

Comment:
I'm not sure if it would make any differences but isn't the DOM element refresh supposed to be at the end (after updating values)?